src/server/config.js

```diff
@@ -58,3 +58,10 @@ function getValue(prop, optional) {
     }
     return config[prop];
 }
+
+function newInstance() {
+    config = {};
+    simulationFilePath = null;
```

Comment:
This is part of the config now. I think you still need to merge master into your branch.